src/components/legalContent/legalTextReader.styles.scss

```
@@ -0,0 +1,256 @@
+		background: transparent;
+		color: var(--m3-on-surface-variant, #444748);
+		cursor: pointer;
+		transition: background-color 150ms ease;
+
+		svg {
+			width: 20px;
+			height: 20px;
+		}
+
+		&:hover {
+			background: color-mix(in srgb, currentColor 8%, transparent);
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)